app/models/attendance.rb

```diff
@@ -0,0 +1,6 @@
+class Attendance < ActiveRecord::Base
+  has_many: :sessions
```

Comment:
`has_many` is actually a method that you pass a symbol to. I could also be written like `has_many(:sessions)` so this is crashing because it doesn't know what `:has_many` the symbol is.